firmware/lucidgloves-firmware/_main.ino

```diff
@@ -25,6 +27,9 @@ void setup() {
 
 void loop() {
   if (comm->isOpen()){
+    #if COMMUNICATION == COMM_BLE
+      BLE.poll();
+    #endif
```

Comment:
This is here because BLE needs polling, otherwise connections will fail.

I don't really like the way I did it here, but I'm not sure whether I should add a poll-like method to ICommunication or just keep it this way.

Reply:
where is `BLE` defined?
and yeah, probably best to add a new method to ICommunication, but let's see what Lucas says.

Reply:
It's a global provided by the ArduinoBLE library.